README.md

```diff
@@ -1,305 +1,241 @@
 
-Insights library can be either loaded via jsDelivr CDN or directly bundled with your application.
-We recommend loading the library by adding the snippet below to all pages where you wish to track
-search analytics.
+The Search Insights library can be either loaded via [jsDelivr CDN](https://www.jsdelivr.com/) or directly bundled with your application.
+We recommend loading the library by adding the snippet below to all pages where you wish to track search analytics.
 
+<!-- prettier-ignore-start -->
 ```html
 <script>
-  !function(e,a,t,n,s,i,c){e.AlgoliaAnalyticsObject=s,e.aa=e.aa||function(){(e.aa.queue=e.aa.queue||[]).push(arguments)},i=a.createElement(t),c=a.getElementsByTagName(t)[0],i.async=1,i.src="https://cdn.jsdelivr.net/npm/search-insights@1.0.0",c.parentNode.insertBefore(i,c)}(window,document,"script",0,"aa");
-
-  // Initialize library
-  aa("init", {
-    appId: "APPLICATION_ID",
-    apiKey: "SEARCH_API_KEY",
-
-    userHasOptedOut: boolean, // Optional. Default: false
-    region: "de" | "us", // Optional. Default: will pick one automatically
-    cookieDuration: number // Optional. In milliseconds. Default: 15552000000ms (6 months)
-  });
-
-  // optional
-  aa("setUserToken", "id-of-user");
+  !function(e,a,t,n,s,i,c){e.AlgoliaAnalyticsObject=s,e.aa=e.aa||function(){(e.aa.queue=e.aa.queue||[]).push(arguments)},i=a.createElement(t),c=a.getElementsByTagName(t)[0],i.async=1,i.src="https://cdn.jsdelivr.net/npm/search-insights@1.2.2",c.parentNode.insertBefore(i,c)}(window,document,"script",0,"aa");
 </script>
 ```
+<!-- prettier-ignore-end -->
 
-### Enabling queryID response from Algolia engine
-
-In order for the Algolia engine to return a queryID on each search request, some special query parameters `clickAnalytics=true` and `enablePersonalization=true` should be set.
+### 2. Initialize the library
 
 ```js
-const search = instantsearch({
-  appId: "APPLICATION_ID",
-  apiKey: "SEARCH_API_KEY",
-  indexName: "INDEX_NAME",
-  searchParameters: {
-    clickAnalytics: true,
-    enablePersonalization: true
-  }
+aa('init', {
+  appId: 'APP_ID',
+  apiKey: 'SEARCH_API_KEY',
 });
+
+// Optional: set the analytics user ID
+aa('setUserToken', 'USER_ID');
 ```
 
-## In the context of search (Click Analytics & A/B testing)
+| Option            | Type          | Default                  | Description                                    |
+| ----------------- | ------------- | ------------------------ | ---------------------------------------------- |
+| **`appId`**       | `string`      | None (required)          | The identifier of your Algolia application     |
+| **`apiKey`**      | `string`      | None (required)          | The search API key of your Algolia application |
+| `userHasOptedOut` | `boolean`     | `false`                  | Whether to exclude users from analytics        |
+| `region`          | `'de' | 'us'` | Automatic                | The DNS server to target                       |
```

Comment:
pipe is interpreted as table break

Reply:
Good catch! Not interpreted the same way on my editor.